### PR TITLE
chore: doc updates, fix build, etc

### PR DESCRIPTION
**Pull request by wstrange**
Update to readme and development notes
add COPY to fix goreleaser error
Convert operator to use SSL!
Add secret agent
Update ldap tests to use SSL

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 FROM gcr.io/distroless/static:nonroot
 WORKDIR /
+COPY ds-operator /ds-operator
 USER nonroot:nonroot
-ENTRYPOINT ["/ds-opertor"]
```
Comment (maxres-fr) [inline build/Dockerfile:4]: :facepalm:

**Review by maxres-fr — APPROVED** [review]

File: pkg/ldap/client_test.go [hunk 2]
```diff
@@ -1,10 +1,10 @@
 // +build integration
-
+// Note the comment above MUST BE ON THE FIRST LINE.
```
Comment (maxres-fr) [inline pkg/ldap/client_test.go:2]: 1 blank space between tag and next comment.

Comment (wstrange) [reply pkg/ldap/client_test.go]: lovely

**Pull request merged**